moped-editor/src/views/projects/projectView/ProjectComponents/mapStyleSettings.js

```diff
@@ -93,7 +93,7 @@ export const MAP_STYLES = {
   "project-lines-muted": {
     isInteractive: false,
     layerProps: {
-      id: "project-lines",
+      id: "project-lines-muted",
```

Comment:
Fixing two ID mismatches here that were causing multiple layers with the same ID and some issues with map click events